<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassManager</class>
 <widget class="QMainWindow" name="PassManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
	background-color:#f8f8f8;
}

QStackedWidget#visualize_stack{
	background-color:#f8f8f8;
}

QLabel{
	color: black;
}

Line{
	color: #3B683B;
}

QPushButton{
	background-color:#486161;
	border-radius: 10px;
	color: #f8f8f8;
	font-weight: bold;
}

QPushButton:hover{
	background-color: #5d8282;
	border: 2px solid #91c3ad;
	color: #f8f8f8;
	font-weight: bold;
}

QTableView{
	background-color: #f8f8f8;
	gridline-color: #cccccc;
	border: 1px solid #dddddd;
	font-size: 12px;
	color: black;
}

QTableView::item{
	padding: 5px;
	border: none;
}

QTableView::item:selected{
	background-color: #486161;
	color: #f8f8f8;
	font-weight: bold;
}

QHeaderView:section{
	background-color: #91c3ad;
	color: white;
	font-weight: bold;
	padding: 5px;
}

QTableCornerButton::section{
	background-color: #91c3ad;
	border: 1px solid #e0e0e0;
}</string>
   </property>
   <widget class="QWidget" name="menu_widget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>131</width>
      <height>571</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>121</width>
       <height>401</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="register_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Registrar</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../assets/icons/add_ad_48dp.svg</normaloff>../assets/icons/add_ad_48dp.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="show_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Visualizar</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../assets/icons/table_eye_48dp.svg</normaloff>../assets/icons/table_eye_48dp.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exit_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Salir</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../assets/icons/exit_to_app_48dp_white_gray.svg</normaloff>../assets/icons/exit_to_app_48dp_white_gray.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MENU</string>
     </property>
    </widget>
    <widget class="QLabel" name="icon_menu_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../assets/icons/menu_48dp_black.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="display_stack">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>511</width>
      <height>441</height>
     </rect>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>20</width>
      <height>441</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
